system/unionfs-fuse/unionfs-fuse.SlackBuild

```diff
@@ -58,7 +58,7 @@ elif [ "$ARCH" = "x86_64" ]; then
   SLKCFLAGS="-O2 -fPIC"
   LIBDIRSUFFIX="64"
 else
-  SLKCFLAGS="-O2"
+  SLKCFLAGS="-O2 -D_FILE_OFFSET_BITS=64"
```

Comment:
:thinking:

Reply:
I see. You might already know this, but in case you don't, there is a block for aarch64 in the official templates:

https://git.slackbuilds.org/templates/commit/?h=15.0&id=20ac57b04874a38a4a1b4aa8d0fa4ec95a5605a4

So feel free to add the block to your scripts, especially if you testing some things on aarch64, which sounds like you are.

Reply:
My aarch64 machine is pretty new, and I just started testing things there.
I'll probably go over all my slackbuilds and make sure everything is ok, and uptodate with the templates.
Over time...
Thanks !